app/templates/sites/new.hbs

```diff
@@ -39,66 +38,67 @@
         </div>
       </:action>
     </PageHeader>
+    {{#if (is-feature-enabled "edit-feature")}}
```

Comment:
Better to avoid this at the route level, redirecting the user to the 404 page

Reply:
As this is no longer accesible if the edit feature is not enabled you can remove this